### DISCUSSION TITLE
How to add free trial users?

### DISCUSSION THREAD

**Discussion by kuzdogan** [Glench/ExtPay#83] [General]
Is it possible to add free trial users? I wasn't able to find and documentation about this and I can see there's a free trial dashboard. 

I'd like all users to start with a 1-day trial. This is the screen users face when they click subscribe and there's no trial option. I tested it and I got charged immediately.

I found [this thread](https://github.com/Glench/ExtPay/discussions/23) but I want 1-day free to be the default membership model and only get charged afterwards (this should be stated explicitly on the payment menu). Is this possible?
<img width="480" alt="image" src="https://user-images.githubusercontent.com/13069972/210107660-d02ec6c2-a839-45f9-8530-bbd62d21be10.png">

Free trial dashboard: 
<img width="883" alt="image" src="https://user-images.githubusercontent.com/13069972/210107883-1d836eb7-b908-49a3-af45-f55094b58d2a.png">

**Comment by Glench** [comment]
Hey, just use `extpay.openTrialPage()`! It's right there in the README.

**Reply by kuzdogan** [reply]
Oh dear... Thanks and sorry
